internal/driver/fetch.go

```diff
@@ -83,6 +83,10 @@ func fetchProfiles(s *source, o *plugin.Options) (*profile.Profile, error) {
 	p.RemoveUninteresting()
 	unsourceMappings(p)
 
+	if len(s.Comment) > 0 {
```

Comment:
`if s.Comment != ""`

Reply:
is there a reason to prefer equality over length?

Reply:
It makes it clear that s.Comment is a string, not a slice.